src/specs/components/page-header-standardised/demo/data--background-image.js

```diff
@@ -9,4 +9,5 @@ module.exports = {
   },
   background_image_url:
     'https://inno-ecl.s3.amazonaws.com/media/examples/example-image.jpg',
+  overlay: 'none',
```

Comment:
mmh..."none" is just a value that we add in the control's select to reset it, it doesn't really exists, indeed in the "docs" comment of the tempalte we only mention "dark" and "light".
If we don't want to use any overlay in the example of the standardised page header we should simply not pass the parameter and in the story file we should use  'none' as the defaultValue of the control.